DXRando/DeusEx/Classes/CCResidentEvilCam.uc

```diff
@@ -0,0 +1,209 @@
+class CCResidentEvilCam extends SecurityCamera;
+
+var DeusExPlayer p;
+var bool Reposition;
+var float cameraMoveTimer;
+
+struct LocationNormal {
```

Comment:
these structs don't appear to be in use, pretty sure you can't use structs across classes like this

Reply:
oh yeah, probably left overs from a previous iteration.  I had copied the structs accross because you can't access them, as you said